zodiac-cli/test/cli/linking/run

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh -eux
+
+. $(dirname $0)/../core/setup.sh
+
+banner TSRP shared object dependencies
+#-------------------------------------
+
+nso="$(objdump -p $TSRP | grep -E '^\s+NEEDED' | wc -l)"
+[ "$nso" = "8" ] || exit 10
```

Comment:
If it could change, it's worth adding a comment explaining the test and possibly enumerating the expected dependencies.